free-3.md

```diff
@@ -6,44 +6,53 @@ good way to meet their needs.
 
 
 ### What's the need?
-_What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help 
-them?_
+People have trouble budgeting out time on their computers. 
 
 
 ### Why a language?
-_Why is a DSL appropriate for your user(s)? How does it address the need?_
+A language would provide enough expressiveness for users to specify their
+schedule and what they should be doing on their computer.
 
 
 ### Why you?
-_What excites you about this idea? How did you come up with it?_
+This is an exciting idea because I think it is a problem most people face
+on their computers: procrastination. I came up with this idea from talking
+to multiple people about their problems with computers.
 
 
 ### Domain
-_Describe the project's domain in five words._
-
+Scheduling and Limiting Screen Time
 
 ### Interface (syntax)
-_How might the user interact with the language? What does programming look 
-like? Why is this the right way to interact with the problem domain?_ 
+I see the syntax as basically describing a schedule for a day and what allowed
+apps/websites would be availiable at certain times. For example I want to 
+accomplish a particular task and I need Safari and Eclipse to do it. Perhaps
+I also want to make sure that I don't play any games instead.
 
+Games = Chess, Pong
+
+Task1 : 2 hours
+	Safari
+	Eclipse
+	not Games
 
-### Operation (semantics)
-_What might happen when a program runs? How does a program interact with the
-user? What kinds of errors might occur, and how might they be communicated to
-the user?_
 
 
+### Operation (semantics)
+When a program runs, it blocks certain programs on your computer from running.
```

Comment:
It's certainly understandable if you don't, but do you have any idea how you might go about doing that?